ocs_ci/ocs/cluster.py

```diff
@@ -1531,6 +1536,25 @@ def is_lso_cluster():
     return config.DEPLOYMENT.get("local_storage", False)
 
 
+def is_flexible_scaling_enabled():
+    """
+    Check if flexible scaling is enabled
+
+    Returns:
+        bool: True if failure domain is "host" and flexible scaling is enabled. False otherwise
+
+    """
+    ocs_storage_cluster = ocp.OCP(
```

Comment:
you can use this function
https://github.com/red-hat-storage/ocs-ci/blob/2c67b247688ac0fac7474cafa7490c98f22e6875/ocs_ci/ocs/resources/storage_cluster.py#L510

Reply:
Fixed